src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,123 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    private static final String INSERT_REQUEST = "INSERT INTO manufacturers(name, country)"
+            + " VALUES (?,?)";
+    private static final String GET_BY_INDEX_REQUEST = "SELECT * FROM manufacturers"
+            + " WHERE ID = (?)";
+    private static final String GET_ALL_REQUEST = "SELECT * FROM manufacturers "
+            + "WHERE is_deleted = false";
+    private static final String UPDATE_REQUEST = "UPDATE manufacturers "
+            + "SET name = (?), country = (?) WHERE id = (?)";
+    private static final String DELETE_REQUEST = "UPDATE manufacturers "
+             + "SET is_deleted = true WHERE id = ?";
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        try (Connection connection = ConnectionUtil.getConnection();
+                     PreparedStatement createFormatStatement = connection
+                             .prepareStatement(INSERT_REQUEST, Statement.RETURN_GENERATED_KEYS)) {
+            createFormatStatement.setString(1, manufacturer.getName());
+            createFormatStatement.setString(2, manufacturer.getCountry());
+            createFormatStatement.executeUpdate();
+            ResultSet generatedKeys = createFormatStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long object = generatedKeys.getObject(1, Long.class);
+                manufacturer.setId(object);
+            }
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Unable to insert format to DB, " + throwables);
```

Comment:
```suggestion
            throw new DataProcessingException("Unable to insert manufacturer to DB, manufacturer: " + manufacturer, throwables);
```

lets make exception messages more informative